go/libkb/upak_loader.go

```diff
@@ -32,7 +32,6 @@ type UPAKLoader interface {
 	LookupUsernameUPAK(ctx context.Context, uid keybase1.UID) (NormalizedUsername, error)
 	LookupUID(ctx context.Context, un NormalizedUsername) (keybase1.UID, error)
 	LookupUsernameAndDevice(ctx context.Context, uid keybase1.UID, did keybase1.DeviceID) (username NormalizedUsername, deviceName string, deviceType string, err error)
-	ListFollowedUIDs(ctx context.Context, uid keybase1.UID) ([]keybase1.UID, error)
```

Comment:
This didn't seem to be correct. Maybe it's because it's loading Unstubbed? I just removed it since it wasn't used anyway.